notebooks/introductory_infilling.ipynb

```diff
@@ -0,0 +1,2572 @@
+       "model       scenario               region variable                                           unit      subannual meta             \n",
+       "AIM/CGE 2.0 ADVANCE_2020_1.5C-2100 World  Emissions|BC                                       Mt BC/yr  0.0       0       2.5998   \n",
+       "                                          Emissions|CH4                                      Mt CH4/yr 0.0       0     133.5733   \n",
+       "                                          Emissions|CH4|AFOLU                                Mt CH4/yr 0.0       0     114.1118   \n",
+       "                                          Emissions|CH4|Energy|Demand|Industry               Mt CH4/yr 0.0       0       0.7735   \n",
+       "                                          Emissions|CH4|Energy|Demand|Residential and Com... Mt CH4/yr 0.0       0       3.7682   \n",
+       "\n",
+       "                                                                                                                           2100  \n",
+       "model       scenario               region variable                                           unit      subannual meta            \n",
+       "AIM/CGE 2.0 ADVANCE_2020_1.5C-2100 World  Emissions|BC                                       Mt BC/yr  0.0       0       2.5771  \n",
+       "                                          Emissions|CH4                                      Mt CH4/yr 0.0       0     133.5872  \n",
+       "                                          Emissions|CH4|AFOLU                                Mt CH4/yr 0.0       0     114.5658  \n",
+       "                                          Emissions|CH4|Energy|Demand|Industry               Mt CH4/yr 0.0       0       0.7577  \n",
+       "                                          Emissions|CH4|Energy|Demand|Residential and Com... Mt CH4/yr 0.0       0       3.5064  \n",
+       "\n",
+       "[5 rows x 22 columns]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sr15_data = pyam.IamDataFrame(SR15_SCENARIOS)\n",
+    "sr15_data.timeseries().head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Crunchers\n",
+    "\n",
+    "Silicone's 'crunchers' are used to determine the relationship between a 'follower variable' and 'lead variable(s)' from a given database. The 'follower variable' is the variable for which we want to generate data e.g. `Emissions|C3F8` while the 'lead variable(s)' is the variable we want to use in order to infer a timeseries of the 'follower variable'. The lead variable is currently always a list with only a single item, e.g. `[Emissions|HFC]` but in future may be able to deal with several lead variables. The follower is always a single string - to infill multiple values in a collective way (e.g. splitting HFC into its various components) see Multiple Infillers. \n",
```

Comment:
For the 3rd sentence, 2nd part it is clear what is meant, but the sentence is not correct

Reply:
rephrased